lib/PlaidClient.js

```diff
@@ -210,6 +211,44 @@ Client.prototype.getTransactions =
     }, options, cb);
   };
 
+// getAllTransactions(String, Date, Date, Object?, Function)
+Client.prototype.getAllTransactions =
+  function(access_token, start_date, end_date, cb) {
+    var transactions = [];
+    var offset = 0, total_transactions = null;
+    var self = this;
+
+    var handleResponse = function(transactionsResponse) {
+      total_transactions = transactionsResponse.total_transactions;
+      transactions.push(...transactionsResponse.transactions);
```

Comment:
we should not use the spread operator, since it assumes that the client is using a version of node that supports it.

even though `node4` is almost at end of life... we should not do this.

consider:

```node
transactions = R.concat(transactions, transactionsResponse.transactions);
```